### PR TITLE
Node.js HTTP request monitoring

### PR DESCRIPTION
**Pull request by miohtama**
This adds hooks for server-side rendering monitoring.

- Show active and stalled requests with [web-top](https://top-framework.readthedocs.io/en/latest/web-top/index.html)
- Uses [web-top-node](https://www.npmjs.com/package/@trading-strategy-ai/web-top-node) package
- Monitoring API key `TOP_WEB_API_KEY` must be set in the secrets file

### CODE REVIEW

File: docs/docker.md [hunk 1]
```diff
@@ -81,9 +81,9 @@ TAG=v4 ; git tag $TAG ; git push origin $TAG
 - [Check that the build completes on Github Actions](https://github.com/tradingstrategy-ai/frontend/actions)
 - [Check container releases on org level](https://github.com/orgs/tradingstrategy-ai/packages)
 
-### Updating the production server
+### Updating the production serverPlain
```
Comment (kenkunz) [inline docs/docker.md:85]: unintended search/replace? (also in line 86 and 163).

File: docs/old-production.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 Deprecated. See [Docker info](./docker.md) instead.
 
-This will run server-side generated (SSR) pages using node.js server.
+This will run serverPlain-side generated (SSR) pages using node.js serverPlain.
```
Comment (kenkunz) [inline docs/old-production.md:5]: same search/replace issue (also line 7)

File: docs/staging.md [hunk 3]
```diff
@@ -6,14 +6,14 @@ Staging environment is password protected to avoid search engine indexing.
 
 The credentials are pinky / angrybird.
 
-This will run server-side generated (SSR) pages using node.js server.
+This will run serverPlain-side generated (SSR) pages using node.js serverPlain.
```
Comment (kenkunz) [inline docs/staging.md:9]: same search/replace issue (also lines 11, 13, 16, 29(

File: docs/theme.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ design system.
 - [Trading Strategy Design System](https://github.com/tradingstrategy-ai/design-system) (phasing-in)
 
 The Bootstrap theme is included as a submodule under `deps/theme`. You can start a
-dev server for the theme:
+dev serverPlain for the theme:
```
Comment (kenkunz) [inline docs/theme.md:10]: same search/replace issue

File: scripts/server-plain.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+		console.log(`URIError: can't decode ${req.url}; forwarding to ${invalidUriPath}`);
+		req.url = invalidUriPath;
+	} finally {
+		handler(req, res);
+	}
+});
+
+serverPlain.listen(port, () => {
+	console.log(`Listening on port: ${port}`);
+});
```
Comment (kenkunz) [inline scripts/server-plain.js:29]: FYI, this file will go away after the SvelteKit upgrades are merged (invalid encoding bug has been fixed).

File: src/routes/about/_Audience.svelte [hunk 6]
```diff
@@ -38,7 +38,7 @@
 			<li>Public open source or private strategies enabled</li>
 
 			<svelte:fragment slot="cta">
-				<Button label="Join our Discord server" icon="discord" href="https://discord.gg/en8tW6MDtw" target="_blank" />
+				<Button label="Join our Discord serverPlain" icon="discord" href="https://discord.gg/en8tW6MDtw" target="_blank" />
```
Comment (kenkunz) [inline src/routes/about/_Audience.svelte:41]: same search/replace issue!

File: src/routes/about/index.svelte [hunk 7]
```diff
@@ -25,7 +25,7 @@
 			<li>Access or white-label our <strong>comprehensive market data sets</strong></li>
 			<li>Gain <strong>access to new markets and investors</strong> directly on-chain, no counter-party risk</li>
 			<li><strong>Create and deploy private strategies</strong> or invest in existing strategies</li>
-			<li><strong>Host private strategies</strong> on your own private server(s)</li>
+			<li><strong>Host private strategies</strong> on your own private serverPlain(s)</li>
```
Comment (kenkunz) [inline src/routes/about/index.svelte:28]: same search/replace issue (also line 60)

File: src/routes/slow-load.svelte [hunk 8]
```diff
@@ -25,7 +25,7 @@ by default. Increment the `?page=n` param to delay by n * 2 seconds.
 		<p>
 			In this page's <code>load</code> function, we waited for {page * 2}
 			seconds to mimick a real-world API call before rendering. You can go back to the home page, which should load instantaneously
-			because it doesn't do much server side. Page 2 (and beyond) will have a longer delay of {(page + 1) * 2} seconds.
+			because it doesn't do much serverPlain side. Page 2 (and beyond) will have a longer delay of {(page + 1) * 2} seconds.
```
Comment (kenkunz) [inline src/routes/slow-load.svelte:28]: same search/replace issue.

File: src/routes/about/_Platform.svelte [hunk 9]
```diff
@@ -28,7 +28,7 @@
 		</li>
 		<li>
 			Market signal analysis and trade entries/exits are executed on the decentralised Trading Strategy Oracle network,
-			which is running off-chain and utilises high server capacity and provessing power not available on “pure” smart
+			which is running off-chain and utilises high serverPlain capacity and provessing power not available on “pure” smart
```
Comment (kenkunz) [inline src/routes/about/_Platform.svelte:31]: same search/replace issue.

**Review by kenkunz — CHANGES_REQUESTED** [review]
It looks like did a global search/replace of `s/server/serverPlain/g` which resulted in a bunch of uninitended changes.

There's a regression of the `deps/theme` submodule commit hash (should be `a44ff27eab` which is latest commit to `master` on the theme repo).

Did you mean to keep `scripts/polka.js` and `scripts/polka-vs-express.js` (since "polka does not work" per `server.js` comment)?

Is the plan to have the top-based server running by default? If so, update last line of `Dockerfile`.

File: .env [hunk 10]
```diff
@@ -3,7 +3,7 @@
 # these values by setting a shell environment variable.
 #
 # Variables prefixed with TS_PUBLIC are imported as run-time configs in
-# config.ts and available server-side or client-side (see hooks/index.ts).
+# config.ts and available serverPlain-side or client-side (see hooks/index.ts).
```
[the remark this answers is not in the document]
Comment (miohtama) [reply .env:6]: Looks like TypeScript "refactor" was in fact "search and replace" under the hood

**Review by kenkunz — APPROVED** [review]
lgtm!

One small question – I noticed the default `Dockerfile` command is the (old) [`scripts/server-plain.js`](https://github.com/tradingstrategy-ai/frontend/blob/16e9f0d13b11bdf9f51160f7499557caa401b603/Dockerfile#L47), while the `docker-compose.yml` command is the (new) [`scripts/server.js`](https://github.com/tradingstrategy-ai/frontend/blob/16e9f0d13b11bdf9f51160f7499557caa401b603/docker-compose.yml#L15). Is that by design?

**Pull request merged**